Frends.Template/Frends.Echo.Execute/Frends.Echo.Execute.csproj

```diff
@@ -0,0 +1,49 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+	  <TargetFramework>net6.0</TargetFramework>
+	  <LangVersion>Latest</LangVersion>
+	  <Version>1.0.0</Version>
+	  <Authors>Frends</Authors>
+	  <Copyright>Frends</Copyright>
+	  <Company>Frends</Company>
+	  <Product>Frends</Product>
+	  <PackageTags>Frends</PackageTags>
+	  <PackageLicenseExpression>MIT</PackageLicenseExpression>
+	  <GenerateDocumentationFile>true</GenerateDocumentationFile>
+	  <Description>Description of the Task</Description>
+	  <PackageProjectUrl>https://frends.com/</PackageProjectUrl>
+	  <RepositoryUrl>https://github.com/FrendsPlatform/Frends.Files/tree/main/Frends.Echo.Execute</RepositoryUrl>
```

Comment:
Can we more or less autogenerate this?

Reply:
We can autogenerate this by using the template powershell command but because this template was just to create new official Tasks IMO the only autogenerated value should be the RepositoryUrl.